### DISCUSSION TITLE
Multiple Bonds

### DISCUSSION THREAD

**Discussion by trunion** [aarons22/homebridge-bond#142] [Q&A] [answered]
Hello, I am trying to configure two Ceiling fans but do not know how exactly to do that.  Can someone help me understand what I am doing wrong ?

[
    {
        "ip_address": "192.168.x.xx",
        "token": "ce24da6e287e46a5"
    }
    {
        "ip_address": "192.168.x.xx”,
        "token": "2b85a9c9c9940813"
    }
]

**Comment by aarons22** [comment, marked as answer]
Looks like you are missing a comma:
```
[
{
"ip_address": "192.168.x.xx",
"token": "ce24da6e287e46a5"
}, <------- right here
{
"ip_address": "192.168.x.xx”,
"token": "2b85a9c9c9940813"
}
]
```
